### DISCUSSION TITLE
Age period filter

### DISCUSSION THREAD

**Discussion by jet-pentest** [projectdiscovery/cvemap#64] [Q&A] [answered]
Hi! The 'age' parameter allows get result with a fixed age of CVE, but period filter are not allowed yet. Like a "Give me all CVEs for 'node.js' in the last 500 days" for filter out potentially old invalid results.

**Comment by ehsandeep** [comment, marked as answer]
@jet-pentest `cvemap -p jira -age '< 500'` this works?

**Comment by jet-pentest** [comment]
Yep, sure! Thanks :)
